pointercrate-user-api/src/endpoints/auth.rs

```diff
@@ -65,6 +128,7 @@ pub async fn login(
 pub async fn invalidate(mut auth: BasicAuth) -> Result<Status> {
     match auth.user {
         AuthenticatedUser::Legacy(legacy) => legacy.invalidate_all_tokens(auth.secret, &mut auth.connection).await?,
+        AuthenticatedUser::OAuth2(oauth) => oauth.invalidate_all_tokens(&mut auth.connection).await? // I have no clue what we'll do here
```

Comment:
mh, this still doesn't work though, because the parameter here is `BasicAuth`, meaning for oauth2 accounts we'll hit `UNAUTHORIZED` way before this point.